src/main/resources/hudson/plugins/ansicolor/Messages.properties

```diff
@@ -20,4 +20,4 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 
-DisplayName = Color ANSI Console Output
+plugin_DisplayName=Color ANSI Console Output

```

Comment:
The exclude rules seems like it could be applied to classes. I found https://github.com/jenkinsci/branch-api-plugin/blob/master/src/findbugs/excludesFilter.xml which has:

```xml
<FindBugsFilter>
  <Match>
    <Class name="~.*\.Messages"/>
  </Match>
</FindBugsFilter>
```

We could also use that here?